liquidctl/driver/hydro_platinum.py

```diff
@@ -268,7 +288,7 @@ def set_color(self, channel, mode, colors, unsafe=None, **kwargs):
         data1 = bytes(itertools.chain(*((b, g, r) for r, g, b in expanded[0:20])))
         data2 = bytes(itertools.chain(*((b, g, r) for r, g, b in expanded[20:])))
         self._send_command(_FEATURE_LIGHTING, _CMD_SET_LIGHTING1, data=data1)
-        self._send_command(_FEATURE_LIGHTING, _CMD_SET_LIGHTING2, data=data2)
+        #self._send_command(_FEATURE_LIGHTING, _CMD_SET_LIGHTING2, data=data2)
```

Comment:
This change may have been accidental, but just in case: this second command is needed because there isn't enough room in the first command to set all (possible) 24 LEDs.

Reply:
yup this was accidental when I was testing something else. 

I also discovered that there is a possible 3rd message for setting LEDs, in case a different fan is being used that has 16 LEDs instead of 4, (mentioned in the  jonasmalacofilho/liquidctl-device-data repository).

Reply:
I missed that part, let me check it again.